SecurityPkg/Library/DxeTcg2PhysicalPresenceLib/DxeTcg2PhysicalPresenceLib.c

```diff
@@ -662,35 +681,55 @@ Tcg2HaveValidTpmRequest  (
       break;
 
     case TCG2_PHYSICAL_PRESENCE_SET_PCR_BANKS:
+      // MS_CHANGE_108842
+      // MSChange [BEGIN] - Do not allow Flags to bypass confirmation in production mode.
+ #ifndef SHIP_MODE

```

Comment:
change the direct usage of SHIP_MODE to a PCD that gets set if SHIP_MODE is enabled